pkgs/development/python-modules/bucketstore/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pythonOlder
+, boto3
+}:
+
+buildPythonPackage rec {
+  pname = "bucketstore";
+  version = "0.2.2";
+  disabled = pythonOlder "3.6";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "00xl1wjpv7417pnsxml22ibkjxakqv96x7gpq2jya3fv7viwp1sd";
+  };
+
+  propagatedBuildInputs = [ boto3 ];
+  doCheck = false;
```

Comment:
The repo contains [tests](https://github.com/jpetrucciani/bucketstore/tree/master/tests). Using from `fetchFromGitHub` would perhaps allow to execute them if they don't need network access. A short comment that stated why the tests are disabled would help for the future.